frontend/src/app/loading.tsx

```
@@ -0,0 +1,12 @@
+import React from "react";
+
+const Loading = () => {
+  return (
+    <div className="flex justify-center items-center flex-col h-screen">
+      <span className="loading loading-ring loading-xl"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)